src/node/cli.ts

```diff
@@ -88,6 +88,7 @@ export interface UserProvidedArgs extends UserProvidedCodeArgs {
   verbose?: boolean
   "app-name"?: string
   "welcome-text"?: string
+  "lng"?: string
```

Comment:
nit: I'd vote for `lang` over `lng`. I think that's more common? Thoughts?

cc @code-asher

Reply:
I would actually prefer we use the existing `--locale` flag since that is what Code uses.

Reply:
Good point!

Reply:
I’m a little worried about whether it affects other places when using the existing `--locale` flag?

Reply:
Hmm...the only place I know is the display language but there may be others.

Reply:
So for example if someone wanted to change the language of code-server all on the command line they would have to do something like:

```
code-server --lng zh-cn --locale zh-cn
```

So it seems unfortunate to have to write the same thing twice and it would not be clear to the user why there are two separate flags.

Reply:
ok. That's right. I will change to use the `--locale` flag.